P/PThreadPool/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PThreadPool"
+version = v"0.0.20191029"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Maratyszcza/pthreadpool.git", "d465747660ecf9ebbaddf8c3db37e4a13d0c9103"),
+    GitSource("https://github.com/Maratyszcza/FXdiv.git", "b742d1143724d646cd0f914646f1240eacf5bd73"),
```

Comment:
Can't this be its own package?

Reply:
It's header-only, so I guess not?

Reply:
Why not?

Reply:
Ah yes - of course it can...

Reply:
Uhm, it appears they need to have the full source code of the dependency including the cmake file. OK, I'll refrain from making any further comments on what they do here.

Reply:
Yeah - I saw that too.

I've left FXdiv as a BuildDependency even though it isn't actually

Reply:
...  - at least it serves to document FXdiv as a dependency in the Pkg-system.

Reply:
err... - it's just a BuildDependency, so: Keep it or remove it?

Reply:
Remove it

Reply:
Done